src/pykeen/models/multimodal/complex_literal.py

```diff
@@ -45,74 +48,42 @@ def __init__(
         embedding_dim: int = 50,
         input_dropout: float = 0.2,
         loss: Optional[Loss] = None,
+        predict_with_sigmoid: bool = False,
         preferred_device: DeviceHint = None,
         random_seed: Optional[int] = None,
         regularizer: Optional[Regularizer] = None,
     ) -> None:
         """Initialize the model."""
         super().__init__(
             triples_factory=triples_factory,
-            embedding_dim=embedding_dim,
+            interaction=LiteralInteraction(
+                base=ComplExInteraction(),
+                combination=ComplExLiteralCombination(
+                    embedding_dim=embedding_dim,
+                    num_of_literals=triples_factory.numeric_literals.shape[-1],
+                    input_dropout=input_dropout,
+                ),
+            ),
+            entity_representations=[
+                EmbeddingSpecification(
+                    embedding_dim=embedding_dim,
+                    initializer=nn.init.xavier_normal_,
+                    dtype=torch.complex64,
+                    # TODO: verify
```

Comment:
this needs to be written up a bit differently - no regularization was reported for this model, right? so it's time to remove it